index.js

```diff
@@ -97,6 +97,11 @@ class PQueue {
 			}
 		});
 	}
+
+	clear() {
+		this.queue._queue = [];
```

Comment:
I was thinking about that as well, but not sure if that's a good idea when the queue has a lot of items. Would there be a downside of just creating a new one?